diff -ruN rethinkdb-2.4.0/mk/support/pkg/openssl.sh rethinkdb-2.4.0-patched/mk/support/pkg/openssl.sh
--- rethinkdb-2.4.0/mk/support/pkg/openssl.sh	2019-12-19 08:45:41.000000000 +0100
+++ rethinkdb-2.4.0-patched/mk/support/pkg/openssl.sh	2020-07-22 08:34:45.208000000 +0200
@@ -1,9 +1,9 @@
 
-version=1.0.2l
+version=1.1.1d
 
 src_url="https://www.openssl.org/source/openssl-$version.tar.gz"
 src_url_backup="ftp://ftp.openssl.org/source/openssl-$version.tar.gz"
-src_url_sha1="b58d5d0e9cea20e571d903aafa853e2ccd914138"
+src_url_sha1="056057782325134b76d1931c48f2c7e6595d7ef4"
 
 pkg_configure () {
     case $($CXX -dumpmachine) in
diff -ruN rethinkdb-2.4.0/src/config/args.hpp rethinkdb-2.4.0-patched/src/config/args.hpp
--- rethinkdb-2.4.0/src/config/args.hpp	2019-12-19 08:45:41.000000000 +0100
+++ rethinkdb-2.4.0-patched/src/config/args.hpp	2020-07-22 08:33:46.396000000 +0200
@@ -100,7 +100,7 @@
 
 // Maximum number of threads we support
 // TODO: make this dynamic where possible
-#define MAX_THREADS                               128
+#define MAX_THREADS                               256
 
 // Ticks (in milliseconds) the internal timed tasks are performed at
 #define TIMER_TICKS_IN_MS                         5
